### PR TITLE
Implemented a simple todo list

### PR DESCRIPTION
**Pull request by pnakongit**
**Pages:**

task_list

task_create

task_update

task_delete

tag_list

tag_create

tag_update

tag_delete

### CODE REVIEW

File: todo_list/models.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class Task(models.Model):
+    content = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField(blank=True, null=True)
+    is_completed = models.BooleanField(default=False)
+    tags = models.ManyToManyField(
+        to=Tag,
+        related_name="tags",
```
Comment (ArsenPidhoretskyi) [inline todo_list/models.py]: wrong related name

File: todo_list/views.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+class TaskDeleteView(generic.DeleteView):
+    model = Task
+    success_url = reverse_lazy("todo_list:index")
+    template_name = "todo_list/confirm_delete.html"
+
+
+class ToggleTaskStatusView(generic.RedirectView):
+    url = reverse_lazy("todo_list:index")
+
+    def get(self, request, *args, **kwargs) -> HttpResponseRedirect:
```
Comment (ArsenPidhoretskyi) [inline todo_list/views.py]: Change objects on get is bad practice, do this in post

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: todo_list/models.py (hunk 1 above)
Comment (pnakongit) [reply todo_list/models.py]: Thank you. Renamed from "tags" to "tasks"

File: todo_list/views.py (hunk 2 above)
Comment (pnakongit) [reply todo_list/views.py]: Thx a lot! Done

**Pull request merged**